### DISCUSSION TITLE
18 gb log files, are they normal?

### DISCUSSION THREAD

**Discussion by matt01y** [ActivityWatch/activitywatch#792] [Q&A] [answered]
Hi all

was doing some routine cleanup, and saw that aw had a fairly large log file.
![Screenshot from 2022-09-16 08-23-53](https://user-images.githubusercontent.com/34476708/190571147-6872e63f-3f07-4441-ae86-07465eb96a89.png)

when i looked into it it just had the same repeating (8148 times to be exact) error in it, namely `BrokenPipeError: [Errno 32] Broken pipe`

i have 2 questions:

1. is it okay to remove this file?
2. do i have to do something to prevent this from happening?

thanks for the help.

**Comment by BelKed** [comment, marked as answer]
1. It's completely fine to remove this file, as it's only a log file
2. Actually, it was fixed recently: https://github.com/ActivityWatch/activitywatch/issues/756#issuecomment-1292126836
